NuGet.Services.Staging.sln

```diff
@@ -30,6 +30,10 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "NuGet.Services.Staging.Unit
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "NuGet.Services.Logging", "src\NuGet.Services.Logging\NuGet.Services.Logging.xproj", "{3A70A370-5ABE-4A53-B5B9-297DB288964E}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "NuGet.Services.Staging.EndToEndTests", "test\NuGet.Services.Staging.EndToEndTests\NuGet.Services.Staging.EndToEndTests.xproj", "{E6859EB8-303B-47C9-AF2B-E62398EDD37B}"

```

Comment:
I would like NuGet.Services.Test.Common to be shared between NuGet projects, so no need for Staging in the name. For example, the TestPackage code in it was copied from Gallery tests, so it will be useful to have a common package. Regarding the other projects, will rename.

---
In reply to: [64173829](https://github.com/NuGet/NuGet.Staging/pull/49#discussion_r64173829) [](ancestors = 64173829)